document/src/test/java/com/yahoo/document/update/TensorModifyUpdateTest.java

```diff
@@ -56,6 +51,12 @@ public void apply_modify_update_operations() {
                 "{{x:0,y:0}:1, {x:0,y:1}:2}", "{{x:0,y:1}:3}", "{{x:0,y:0}:1,{x:0,y:1}:5}");
         assertApplyTo("tensor(x[1],y[2])", Operation.MULTIPLY,
                 "{{x:0,y:0}:3, {x:0,y:1}:2}", "{{x:0,y:1}:3}", "{{x:0,y:0}:3,{x:0,y:1}:6}");
+        assertApplyTo("tensor(x{},y[2])", Operation.REPLACE,
```

Comment:
Follows the same argument as remove() and merge(). The detailed tests should be moved to the tensor library. Only the integration in applyTo() should be tested here.